src/sympc/api.py

```diff
@@ -175,6 +176,7 @@
         "sympc.tensor.ReplicatedSharedTensor.clone",
         "sympc.tensor.ReplicatedSharedTensor",
     ),
+    ("sympc.tensor.ReplicatedSharedTensor.shape", "syft.lib.python.Tuple"),
```

Comment:
Q: I think you can remove this, right? (since now you call shape from a "local" context)

Reply:
Nope, this was actually the fix for the error where ReplicatedShareTensor was being passed as a shape to PRZS. Also in the future, we might need it anyway?

Reply:
Yep, but we should add it when we need it.
I am not sure if now we are using it. If you remove it, it should still work (this allowlist is for being able to call shape on a remote RST)

Reply:
Yeah, dint need this.